jenkins/infra/terraform/locals.tf

```diff
@@ -0,0 +1,15 @@
+locals {
+  ip_subnet_mapping = [{
+    subnet = "subnet-0a496f50"
+    eip    = "eipalloc-0817cbaf1e9476916"
```

Comment:
Indentation seems off here – did you run `terraform fmt`?

Reply:
hmm, yes I did indeed